tests/functional/customer/test_notification.py

```diff
@@ -13,3 +15,9 @@ def test_can_see_them_in_page_header(self):
         homepage = self.app.get('/', user=self.user)
         self.assertTrue('num_unread_notifications' in homepage.context)
         self.assertEqual(1, homepage.context['num_unread_notifications'])
+
+        path = reverse('customer:notifications-inbox')
```

Comment:
Please make this a separate test rather than part of the previous one.

Reply:
You also need to add a second test to check that a notification is marked as read when it is viewed in the detail view.